Document tunnel usage in github actions (#231)
saucelabs/sauce-docs: docs/testrunner-toolkit/integrations/github-actions.md

## Create the Test Job

> In the examples below, we illustrate the different run modes that `saucectl` has: Docker and the Sauce Cloud. They determine where the tests are executed. Docker means that tests are executed locally in a container, while Sauce refers to the Sauce Cloud (i.e. Sauce Labs infrastructure).

Reviewer: ```suggestion
In the examples below, we illustrate the different run modes that `saucectl` has: __Docker__ and the __Sauce Cloud__— both determine where tests execute. Docker refers to executing tests locally in a container, while Sauce refers to executing tests on Sauce Cloud (i.e. Sauce Labs infrastructure).
```

Reviewer: tried to remove passive voice.

> If you are running your tests on the Sauce Cloud, you will likely require a tunnel back to where your application is running. A tunnel will enable the remote browser to access your local network.

Reviewer: ```suggestion
If you run your tests on the Sauce Cloud, you will likely require a tunnel back to where your application is running. A tunnel enables the remote browser to access your local network.
```

Reviewer: some passive voice is still ok in this scenario

> For this, we are going to use [Sauce Connect](secure-connections/sauce-connect.md).

Reviewer: ```suggestion
For this, we are going to use [Sauce Connect](/secure-connections/sauce-connect.md).
```

> For more detailed information on setting event-driven actions and jobs, please visit the [GitHub Action documentation](https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/introduction-to-github-actions#the-components-of-github-actions).

<Tabs
  defaultValue="Docker"
  values={[
    {label: 'Docker', value: 'Docker'},
    {label: 'Sauce Cloud', value: 'Sauce Cloud'},
  ]}>
  
<TabItem value="Docker">

```yaml reference
https://github.com/saucelabs/testrunner-toolkit/blob/master/.github/workflows/tests.yml#L93-L114
```

</TabItem>
<TabItem value="Sauce Cloud">

```yaml reference
https://github.com/saucelabs/sauce-docs/blob/master/.github/workflows/deploy.yml#L79-L93
```

</TabItem>
</Tabs>

> > You can reference our example workflows [here](https://github.com/saucelabs/testrunner-toolkit/tree/master/.github/workflows) and [here](https://github.com/saucelabs/sauce-docs/blob/master/.github/workflows).

Reviewer: I think the GH plugin already provides a "see the full example on GitHub" at the bottom of the code box.
